kickoff/test/views/test_json.py

```diff
@@ -148,20 +149,33 @@ def testTBBetaBuilds(self):
     def testFirefoxVersions(self):
         config.CURRENT_ESR = "2"
         config.ESR_NEXT = "38"
-        config.AURORA_VERSION = "23.0a2"
-        config.NIGHTLY_VERSION = "24.0a1"
         ret = self.get(BASE_JSON_PATH + '/firefox_versions.json')
         versions = json.loads(ret.data)
 
         self.assertEquals(ret.status_code, 200)
         self.assertEquals(versions['FIREFOX_ESR_NEXT'], "38.1.0esr")
         self.assertEquals(versions['FIREFOX_ESR'], "2.0.2esr")
         self.assertEquals(versions['LATEST_FIREFOX_RELEASED_DEVEL_VERSION'], "3.0b3")
-        self.assertEquals(versions['FIREFOX_AURORA'], "23.0a2")
-        self.assertEquals(versions['FIREFOX_NIGHTLY'], "24.0a1")
         self.assertEquals(versions['LATEST_FIREFOX_VERSION'], '2.0')
         self.assertEquals(versions['LATEST_FIREFOX_OLDER_VERSION'], "3.6.28")
         self.assertEquals(versions['LATEST_FIREFOX_DEVEL_VERSION'], "3.0b3")
+
+        # We don't use assertRegex because it needs Python 2.7.
+        # We limit the check on the version number to 2 digits intentionally.
+        # Failing on a typo such as 530.a1 is more of a worry than having to
+        # update this test when we reach Firefox 100.
+        self.assertNotEquals(
+            re.match(r'\d\d\.0a1', versions['FIREFOX_NIGHTLY']),
```

Comment:
If you look at the comment above, I had limited that on purpose because my goal is to limit manual errors such as typing 533.0a1 instead of 53.0a1, so yes that means updating the test when we reach Firefox 100 but it could catch manual input errors in the meantime that \d+ wouldn't and actually \d+ wouldn't catch 10000000.a1 as an error :)